extensions/resteasy-reactive/rest-client-microprofile/deployment/src/test/java/io/quarkus/resteasy/reactive/client/registerclientheaders/HeaderPassingClient.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.resteasy.reactive.client.registerclientheaders;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterRestClient
+@RegisterClientHeaders
+public interface HeaderPassingClient {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
A bit off topic, but is this required? If so we should look at using the same defaults we use for server side REST (i.e. text/plain for String, JSON for everything else).

Reply:
no, it shouldn't be. I'll remove it